data/abilities/collection/30a8cf10-73dc-497c-8261-a64cc9e91505.yml

```diff
@@ -11,22 +11,24 @@
     linux:
       sh:
         command: |
-          tar -C #{host.dir.staged} -czf - . | gpg -c --pinentry-mode=loopback --passphrase #{host.archive.password} > #{host.dir.staged}.tar.gz.gpg && echo #{host.dir.staged}.tar.gz.gpg
+          tar -C #{file.dir} -czf - . | gpg -c --pinentry-mode=loopback --passphrase #{custom.archive_password} > #{file.dir}.tar.gz.gpg && echo #{file.dir}.tar.gz.gpg
```

Comment:
are we no longer differentiating between staging directories and normal directories?

Reply:
Per the confluence page, I don't believe we are differentiating between staging and normal directories. It might be worthwhile to revisit this and use `file.staged_dir` instead

Reply:
It would be helpful to distinguish between regular discovered directories and directories that we want to take certain actions on, like compression. This could be a good use-case for the sub-minor tag, like `file.dir.staged`